### ISSUE TITLE
About the dropout layer

### ISSUE THREAD

**Issue by haojiubujian91** [VDelv/EEGLearn-Pytorch#18]
The author used two dropout layers in five models in the original paper (Learning representations from EEG with deep recurrent-convolutional neural networks). But I didn’t see it in your code. Did you design it this way on purpose? Looking forward to your reply.
